Desktop/data/importers/excel/excel.h

```diff
@@ -0,0 +1,54 @@
+//
+// Copyright (C) 2013-2024 University of Amsterdam
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 2 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+//
+
+#ifndef EXCEL_H
+#define EXCEL_H
+
+#include <vector>
+#include <string>
+#include <stdint.h>
+
+#include <freexl.h>
```

Comment:
So here you include "freexl.h" from the include paths, but why then add `__freexl.h' ?